starboard/android/shared/media_decoder.cc

```diff
@@ -159,6 +162,9 @@ MediaDecoder::~MediaDecoder() {
   }
 
   if (is_valid()) {
+    // Clear the native media codec bridge handle in java layer to prevent
```

Comment:
`media_codec_bridge_->Flush()` is after this `media_codec_bridge_->Stop()`, which makes Cobalt fails to access MediaCodec.